lib/importers/rating_importer.rb

```diff
@@ -55,7 +55,8 @@ def self.update_ratings(article_groups)
   ####################
   def self.update_article_ratings(articles, ratings)
     articles.each do |article|
-      article.rating = ratings[article.title]
+      article.rating_all = ratings[article.title]
```

Comment:
(Continuing from IRC)... I see your point about how the use of article.rating is not completely abstracted for use with the ArticleHelper method... we pass three different versions of the rating data to the frontend, via the course articles.json endpoint: 'rating' (the one from the model directly), 'pretty_rating' (from `ArticleHelper#display_rating`) and 'rating_num' (from `ArticleHelper#rating_priority`). However, I think it makes more sense to save only the full ratinig, and deal with whatever display issues come up later on. For the rating docs, the best solution will probably be to add more docs for ratings that aren't currently supported.